### PR TITLE
Pandas update

### PR DESCRIPTION
**Pull request by cJames9**
Addressed many deprecations from Pandas since version 0.20.0, up to 1.4.0:

- .ix replaced by .loc and .iloc
- implemented datetime, replacing pd.datetime
- pandas.util.testing relocated to pandas.testing
- Int64Index is deprecated in version 1.4.0, replaced by Index with dtype Int64
- removed HistoricalDataResponse.as_panel, since pd.Panel has been replaced by pd.MultiIndex, which is supported by HistoricalDataResponse.as_frame
- implemented dateutil.relativedelta, replacing pd.datetools

Thus, I've set pandas 1.4.0 in requirements.txt.

### CODE REVIEW

File: tia/bbg/example.py [hunk 1]
```diff
@@ -44,4 +46,4 @@ def banner(msg):
     #
     # - Retrieve an fx vol surface:  BbgReferenceDataRequest('eurusd curncy', 'DFLT_VOL_SURF_MID')
     # - Retrieve a fx forward curve:  BbgReferenceDataRequest('eurusd curncy', 'FWD_CURVE')
-    # - Retrieve dividends:  BbgReferenceDataRequest('csco us equity', 'BDVD_PR_EX_DTS_DVD_AMTS_W_ANN')
\ No newline at end of file
+    # - Retrieve dividends:  BbgReferenceDataRequest('csco us equity', 'BDVD_PR_EX_DTS_DVD_AMTS_W_AN'N')
```
Comment (PaulMest) [inline tia/bbg/example.py]: Is this a typo? Looks like there's an extra apostrophe here.

File: tia/bbg/v3api.py [hunk 2]
```diff
@@ -254,9 +254,6 @@ def __init__(self, request):
     def on_security_complete(self, sid, frame):
         self.response_map[sid] = frame
 
-    def as_panel(self):
-        return pd.Panel(self.response_map)
```
Comment (PaulMest) [inline tia/bbg/v3api.py]: Consider adding something like this?
```python
def as_panel(self):
    raise NotImplementedError('Pandas.Panel was removed from the library: https://pandas.pydata.org/pandas-docs/version/0.23/generated/pandas.Panel.html.')
```

File: tia/rlab/table.py [hunk 3]
```diff
@@ -134,7 +134,7 @@ def span_iter(series):
                 yield sorted.index[li], sorted.index[i + 1]
         else:
             li = i + 1
-    raise StopIteration
+    #raise StopIteration
```
Comment (PaulMest) [inline tia/rlab/table.py:137]: What did this do before? Why comment this out?

File: README.md [hunk 4]
```diff
@@ -5,19 +5,23 @@ There are now community contributions to address deprecation warnings in Pandas.
 
 ### Install tia with Python 3 changes + latest community contributions
 ```
-$ pip install git+git://github.com/PaulMest/tia.git#egg=tia
+pip install https://github.com/cjames9/tia/archive/master.zip
```
Comment (PaulMest) [inline README.md]:
```suggestion
pip install https://github.com/PaulMest/tia/archive/master.zip
```

File: README.md [hunk 5]
```diff
@@ -5,19 +5,23 @@ There are now community contributions to address deprecation warnings in Pandas.
...
+```
+or
+```
+$ pip install git+https://github.com/cjames9/tia.git#egg=tia
 ```
 or put one of these into your `requirements.txt` file
 ```
 # Using git/SSH
-git+git://github.com/PaulMest/tia.git#egg=tia
+git+git://github.com/cjames9/tia.git#egg=tia
```
Comment (PaulMest) [inline README.md]:
```suggestion
git+git://github.com/PaulMest/tia.git#egg=tia
```

File: README.md [hunk 6]
```diff
@@ -5,19 +5,23 @@ There are now community contributions to address deprecation warnings in Pandas.
...
 
 # Using https
-git+https://github.com/PaulMest/tia.git#egg=tia
+git+https://github.com/cjames9/tia.git#egg=tia
```
Comment (PaulMest) [inline README.md]:
```suggestion
git+https://github.com/PaulMest/tia.git#egg=tia
```

File: README.md [hunk 7]
```diff
@@ -5,19 +5,23 @@ There are now community contributions to address deprecation warnings in Pandas.
...
+pip install https://github.com/cjames9/tia/archive/master.zip
 ```
 or
 ```
-$ pip install git+https://github.com/PaulMest/tia.git#egg=tia
+$ pip install git+git://github.com/cjames9/tia.git#egg=tia
+```
+or
+```
+$ pip install git+https://github.com/cjames9/tia.git#egg=tia
```
Comment (PaulMest) [inline README.md]:
```suggestion
$ pip install git+https://github.com/PaulMest/tia.git#egg=tia
```

File: README.md [hunk 8]
```diff
@@ -5,19 +5,23 @@ There are now community contributions to address deprecation warnings in Pandas.
 
 ### Install tia with Python 3 changes + latest community contributions
 ```
-$ pip install git+git://github.com/PaulMest/tia.git#egg=tia
+pip install https://github.com/cjames9/tia/archive/master.zip
 ```
 or
 ```
-$ pip install git+https://github.com/PaulMest/tia.git#egg=tia
+$ pip install git+git://github.com/cjames9/tia.git#egg=tia
```
Comment (PaulMest) [inline README.md]:
```suggestion
$ pip install git+git://github.com/PaulMest/tia.git#egg=tia
```

**Comment by PaulMest** [comment]
Hi there. Thanks for this PR. I haven't personally used this library in several years. But I've reviewed what you've done and am happy to consider merging it if you can respond to the comments I made.

**Comment by cJames9** [comment]
Hi Paul. Actually, I intended to create this PR in my own repo, I've cloned yours and fixed a few deprecations, and have been using it in my job. Sorry, but I couldn't find your comments, where are they?

**Review by PaulMest — CHANGES_REQUESTED** [review]

**Comment by PaulMest** [comment]
> Actually, I intended to create this PR in my own repo

That makes a lot more sense! 😅

File: tia/rlab/table.py (hunk 3 above)
Comment (cJames9) [reply tia/rlab/table.py:137]: This one I couldn't understand. Everytime I created a table, I would receive a StopIteration exception. After I commented this line out, the code executed correctly, and there was no errors on the tables.

File: README.md (hunk 4 above)
Comment (cJames9) [reply README.md]: Totally agree. As I've said, this PR was intended to be on my personal repo, so I've adjusted this to reflect my branches.

File: tia/bbg/example.py (hunk 1 above)
Comment (cJames9) [reply tia/bbg/example.py]: Yes, this is a typo. I've sent another commit fixing this.

File: tia/bbg/v3api.py (hunk 2 above)
Comment (cJames9) [reply tia/bbg/v3api.py]: Good one.

**Comment by cJames9** [comment]
> > Actually, I intended to create this PR in my own repo
> 
> That makes a lot more sense! 😅

Sorry about the confusion 😅 Still, your repo is still much more popular, I hope all of this can help anyone who uses it.

**Comment by PaulMest** [comment]
On behalf of all of the quants and traders who begrudgingly grep through countless poorly written finance API docs and fiddle with Python package versions so that they can eventually get their "real work" done, I thank you for your contribution @cJames9! 🙏🏼

**Pull request merged**